tools/tpm2_createpolicy.c

```diff
@@ -320,27 +160,23 @@ int execute_tool(int argc, char *argv[], char *envp[], common_opts_t *opts,
     (void) opts;
     (void) envp;
 
-    createpolicypcr_ctx ctx = { 
-        .sapi_context = sapi_context, 
-        .policy_session = NULL,
-        .policy_digest = TPM2B_TYPE_INIT(TPM2B_DIGEST, buffer),
-        .pcr_index = 0,
-        .max_supported_pcrs = 0,
-        .hash_alg = TPM_ALG_SHA1,
-        .policyfile = NULL,
-        .raw_pcr_file = NULL,
-        .pcr_flags.policy_type_pcr_flag = false,
-        .pcr_flags.pcr_index_flag = false,
-        .pcr_flags.raw_pcr_flag = false
+    CREATE_POLICY_CTX pctx = {
+        .sapi_context = sapi_context,
+        .common_policy_options = {
+            .policy_session = NULL,
+            .policy_session_type = TPM_SE_TRIAL,
+            .policy_digest = TPM2B_TYPE_INIT(TPM2B_DIGEST, buffer),
+            .policy_digest_hash_alg = TPM_ALG_SHA256,
```

Comment:
You're requiring the caller to magically know the right defaults for this. I would put this into a macro in tpm2_policy.h
```
...
.common_policy_options = TPM2_COMMON_POLICY_INIT,
},
```